dataikuapi/dss/analysis.py

```diff
@@ -225,6 +225,44 @@ def create_clustering_ml_task(self,
             mltask.wait_guess_complete()
         return mltask
 
+    def create_timeseries_forecasting_ml_task(self, input_dataset, target_variable,
+                                              time_variable=None,
+                                              timeseries_identifiers=None,
+                                              guess_policy="TIMESERIES_DEFAULT",
+                                              wait_guess_complete=True):
+
+        """Creates a new prediction task in a new visual analysis lab for a dataset.
```

Comment:
```suggestion
        """Creates a new timeseries forecasting task in this visual analysis lab for a dataset.
```

\+ should also fix the "a new visual analysis lab" → "this visual analysis lab" for clustering above